Clients/GUI/Pages/SignIn.razor

```diff
@@ -0,0 +1,71 @@
+﻿@page "/singin"
+@using Authentication
+@using ViewModels
+@using DTO
+@using Scripts
+@inject AuthStateProvider AuthenticationProvider
+@inject NavigationManager NavigationManager
+
+<h3 style="color: red; display: @(isHidden ? "none" : "block")"> Wrong login or password</h3>
+<div class="col-12">
+    <h3><b>Sign In</b></h3>
+    <hr />
+    <EditForm EditContext="@_editContext" OnValidSubmit="@TrySignIn">
+        <DataAnnotationsValidator />
+        <div class="form-group">
+            <label for="Email">Email</label>
+            <InputText class="form-control col-3" @bind-Value="userInput.Email" placeholder="example@gmail.com" />
+            <ValidationMessage For="() => userInput.Email" />
+        </div>
+        <br />
+        <div class="form-group">
+            <label for="Password">Password</label>
+            <InputText class="form-control col-3" type="Password" @bind-Value="userInput.Password" placeholder="*****" />
+            <ValidationMessage For="() => userInput.Password" />
+        </div>
+
+        <input type="submit" class="form-control col-1 button btn-primary" value="Sign in" />
+
+    </EditForm>
+</div>
+
+
+@code {
+    [CascadingParameter]
+    private Task<AuthenticationState> authState { get; set; }
+    private SignInViewModel userInput;
+    private EditContext _editContext;
```

Comment:
we have decided that name private variables with camelCase

Reply:
Agree

Reply:
Fixed